.github/workflows/ci-aarchxx.yml

```diff
@@ -75,6 +75,30 @@ jobs:
       env:
         DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: yes
 
+    - name: Send failure mail to dynamorio-devs
+      if: failure() && github.ref == 'refs/heads/master'
+      uses: dawidd6/action-send-mail@v2
+      with:
+        server_address: smtp.gmail.com
+        server_port: 465
+        username: ${{secrets.DYNAMORIO_NOTIFICATION_EMAIL_USERNAME}}
```

Comment:
@derekbruening Can you please rename the Secret vars? It is better to have a DYNAMORIO_ prefix on these variables to reduce chances of conflict with a forked repo's Secret vars. See comment on  main PR thread for details.

Reply:
They can't be renamed.  I created new ones with the new names and deleted the old ones.